app/portainer/components/forms/git-form/git-form-additional-files-panel/git-form-additional-file-item/index.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+import controller from './git-form-additional-file-item.controller.js';
+
+export const gitFormAdditionalFileItem = {
+  templateUrl: './git-form-additional-file-item.html',
+  controller,
+
+  bindings: {
+    variable: '<',
+    index: '<',
+
+    onChange: '<',
+    onRemove: '<',
+  },
+};
+
+angular.module('portainer.app').component('gitFormAdditionalFileItem', gitFormAdditionalFileItem);
```

Comment:
this can be removed as this component is defined at https://github.com/portainer/portainer/blob/f7f26aa81d0e191dbf51bb76659dfe1b885f2bc2/app/portainer/components/forms/git-form/index.js#L17

Reply:
I think they are 2 different components.

Reply:
right. but this one (additionalFileItem) is used in `gitFormAdditionalFilesPanel`, so probably should be deinfed on the same place, no?

Reply:
yes, updated